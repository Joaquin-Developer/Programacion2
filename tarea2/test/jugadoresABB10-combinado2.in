crearArbolJugadores

# -- primer jugador Persona1 --
crearJugada
1 1 2 3 4
agregarEnPartida
crearJugador 1 1 Uruguay Persona1
insertarJugadorArbolJugadores


imprimirArbolJugadores

removerArbolJugadores 1

estaEnArbolJugadores 1
esVacioArbolJugadores

# -- segundo jugador Persona5 --
crearJugada
1 1 2 3 3
agregarEnPartida
crearJugador 5 55 Argentina Persona5
insertarJugadorArbolJugadores

# -- tercer jugador Persona10 --
crearJugada
1 1 2 3 3
agregarEnPartida
crearJugador 10 10 Argentina Persona10
insertarJugadorArbolJugadores

# -- cuarto jugador Persona20 --
crearJugada
1 4 4 4 4
agregarEnPartida
crearJugador 20 20 Argentina Persona20
insertarJugadorArbolJugadores

# -- quinto jugador Persona15 --
crearJugada
1 2 2 2 2
agregarEnPartida
crearJugador 15 15 Argentina Persona15
insertarJugadorArbolJugadores


aplanarArbolJugadores

removerArbolJugadores 10
estaEnArbolJugadores 10

mayoresEdadArbolJugadores 10

removerArbolJugadores 20

esPerfectoArbolJugadores

# -- quinto jugador Persona2 --
crearJugada
1 2 2 2 2
agregarEnPartida
crearJugador 2 22 Bolivia Persona2
insertarJugadorArbolJugadores

esPerfectoArbolJugadores

liberarArbolJugadores

Fin







