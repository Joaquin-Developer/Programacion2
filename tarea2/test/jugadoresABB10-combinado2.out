1>2>#  -- primer jugador Persona1 --.
3>4>5>6>7>Jugador 1:  Persona1, Uruguay, 1 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Abajo 
8>9>La jugador con id 1 NO está en el árbol de jugadores.
10>El árbol de jugadores está vacío.
11>#  -- segundo jugador Persona5 --.
12>13>14>15>16>#  -- tercer jugador Persona10 --.
17>18>19>20>21>#  -- cuarto jugador Persona20 --.
22>23>24>25>26>#  -- quinto jugador Persona15 --.
27>28>29>30>31>Comando no reconocido.
32>33>La jugador con id 10 NO está en el árbol de jugadores.
34>Jugador 5:  Persona5, Argentina, 55 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Arriba 
Jugador 15:  Persona15, Argentina, 15 años
Jugada: 1 
Movimientos: Izquierda Izquierda Izquierda Izquierda 
Jugador 20:  Persona20, Argentina, 20 años
Jugada: 1 
Movimientos: Abajo Abajo Abajo Abajo 
35>36>El árbol de jugadores NO es perfecto.
37>#  -- quinto jugador Persona2 --.
38>39>40>41>42>El árbol de jugadores es perfecto.
43>44>Fin.
